test/fixtures/stacks/catalog/usecase/basic.yaml

```
@@ -5,3 +5,5 @@ components:
         component: target
       vars:
         enabled: true
+        organization_name: test
+        region: us-east-2
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)